doc/source/programs/gdal_raster_pixelinfo.rst

```diff
@@ -0,0 +1,238 @@
+.. _gdal_raster_pixelinfo:
+
+================================================================================
+``gdal raster pixelinfo``
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Return information on a pixel of a raster dataset
+
+.. Index:: gdal raster pixelinfo
+
+Synopsis
+--------
+
+.. program-output:: gdal raster pixelinfo --help-doc
+
+Description
+-----------
+
+:program:`gdal raster pixelinfo` provide a mechanism to query information about
+a pixel given its location in one of a variety of coordinate systems.
+
+It supports outputting either as GeoJSON or CSV.
+
+The following items will be reported (when known):
+
+- Input coordinates
+- Input coordinates converted to column, line
+- Pixel value per selected band(s), with unscaled value when offset/scale is associated to the band
```

Comment:
On the one hand, this is convenient (don't have to specify if you want unscaling or not). On the other hand, it makes the output less predictable. (Imagine tossing a bunch of rasters at this and then suddenly the desired output is in `band_1_unscaled_value` instead of `band_1_value`). For example, I have a bunch of temperature data downloaded from Copernicus. The files I downloaded a couple of years ago scale the temperature into an int16, the newer ones just use a float. I'm not sure what the use case for reading scaled values is, so I would unscale by default into `band_1_value`.

Reply:
band_1_raw_value and band_1_unscaled_value now systematically emitted